config/locales/de.yml

```diff
@@ -78,6 +78,7 @@ de:
         intro: |
           Um per Telegram teilzunehmen, folgen Sie bitte diesen vier Schritten:
         steps:
+          - Wenn Sie bereits Telegramm haben, können Sie sich anmelden, indem Sie auf die Schaltfläche unten klicken, um uns zu autorisieren, Ihnen Nachrichten zu senden.
```

Comment:
please @FrauCsu can you give me what you want this step to say actually? This is a placeholder.